layers/chassis/chassis_modification_state.h

```diff
@@ -42,27 +42,33 @@ struct CreateShaderModule {
     spirv::StatelessData stateless_data;
 };
 
+struct ShaderObjectInstrumentationMetadata {
```

Comment:
```suggestion
struct ShaderObjectInstrumentationData {
```
Since this holds `instrumented_spirv`, it does not only hold _Meta_ data, but really instrumentation data. Yesterday it was truly "Metadata", here not so much : 3